torch_frame/data/dataset.py

```diff
@@ -212,8 +218,8 @@ def _get_mapper(self, col: str) -> TensorMapper:
             return CategoricalTensorMapper(index)
         elif stype == torch_frame.multicategorical:
             index, _ = self.col_stats[col][StatType.MULTI_COUNT]
-            return MultiCategoricalTensorMapper(index,
-                                                sep=self.col_to_sep[col])
+            return MultiCategoricalTensorMapper(
+                index, sep=self.col_to_sep.get(col, None))
```

Comment:
`sep = None` seems wrong.

Reply:
Discussed over slack. We agreed that it's weird for customers to specify a dummy string when the multicategorical column is a list. The `None` case refers to the case just mentioned. No strings splitting will occur in case of None.